cumulusci/cli/cci.py

```diff
@@ -722,6 +722,33 @@ def project_dependencies(runtime):
         click.echo(line)
 
 
+@project.command(
+    name="doc", help="Exports RST format documentation for all project tasks"
+)
+@pass_runtime(require_project=False)
+def project_doc(runtime):
+
+    try:
+        os.mkdir("./docs")
+    except Exception:
+        pass  # skipping as not to freak end user out since directory exists.
+    finally:
+        project_config = runtime.project_config.config
+        project_name = project_config["project"]["name"].capitalize()
+
+        with open("./docs/project_tasks.rst", "w") as f:
+            f.write("==========================================\n")
+            f.write(f"{project_name} Tasks Reference\n")
+            f.write("==========================================\n")
+            f.write("\n")
+            for name, options in project_config["tasks"].items():
+                task_config = TaskConfig(options)
+                doc = doc_task(name, task_config)
```

Comment:
Put this inside the `if` so we aren't spending time building docs for tasks that we're not going to include in the output.